RoslynAnalyzers.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26228.0
+VisualStudioVersion = 15.0.26424.2
```

Comment:
Not really. VS will just update it to whatever version you're running when you open the solution anyway.